examples/transformers_custom_handler/download_glue_data.py

```diff
@@ -0,0 +1,141 @@
+''' Script for downloading all GLUE data.
+
+Note: for legal reasons, we are unable to host MRPC.
+You can either use the version hosted by the SentEval team, which is already tokenized, 
+or you can download the original data from (https://download.microsoft.com/download/D/4/6/D46FF87A-F6B9-4252-AA8B-3604ED519838/MSRParaphraseCorpus.msi) and extract the data from it manually.
+For Windows users, you can run the .msi file. For Mac and Linux users, consider an external library such as 'cabextract' (see below for an example).
+You should then rename and place specific files in a folder (see below for an example).
+
+mkdir MRPC
+cabextract MSRParaphraseCorpus.msi -d MRPC
+cat MRPC/_2DEC3DBE877E4DB192D17C0256E90F1D | tr -d $'\r' > MRPC/msr_paraphrase_train.txt
+cat MRPC/_D7B391F9EAFF4B1B8BCE8F21B20B1B61 | tr -d $'\r' > MRPC/msr_paraphrase_test.txt
+rm MRPC/_*
+rm MSRParaphraseCorpus.msi
+
+1/30/19: It looks like SentEval is no longer hosting their extracted and tokenized MRPC data, so you'll need to download the data from the original source for now.
+2/11/19: It looks like SentEval actually *is* hosting the extracted data. Hooray!
+'''
+
+import os
+import sys
+import shutil
+import argparse
+import tempfile
+import urllib.request
+import zipfile
+
+TASKS = ["CoLA", "SST", "MRPC", "QQP", "STS", "MNLI", "SNLI", "QNLI", "RTE", "WNLI", "diagnostic"]
+TASK2PATH = {"CoLA":'https://firebasestorage.googleapis.com/v0/b/mtl-sentence-representations.appspot.com/o/data%2FCoLA.zip?alt=media&token=46d5e637-3411-4188-bc44-5809b5bfb5f4',
```

Comment:
@MFreidank Instead of hardcoding all the taskpath downloads, it will be better to use the pretrained models in the example. That way the example can be generic and one will not have to keep updating if the download path changes in future. Please see what can be done for generalizing the example

Reply:
@chauhang Thanks for your feedback - very valid point. I've significantly simplified my example, I am loading a pretrained checkpoint using huggingface for a distilBERT model pre-trained on the SST-2 binary sentiment analysis task. 
This alleviates any need to hardcode paths or fine-tune any model, making the example significantly more accessible.

Model snapshot I am using can be found [here](https://huggingface.co/distilbert-base-uncased-finetuned-sst-2-english).

More information on the SST-2 task the model was finetuned for can be found [here](https://nlp.stanford.edu/sentiment/).